DesignSafe-CI/DS-User-Guide
# fix: GH-56 delete manual heading permalinks

## Overview

Delete manual permalinks for headings (cuz #58 will add automatic ones).

## Related

- GH-56
- coupled with #58
- mimics https://github.com/TACC/TACC-Docs/pull/38

## Changes

- **deleted** links from headings (but kept `id`'s)

## Testing

1. Open https://pprd.docs.tacc.utexas.edu/curating/#policies.
2. Hover on "Mission".
3. Verify it is not a link.
4. Open https://pprd.docs.tacc.utexas.edu/curating/#mission.
5. Verify page scrolls to "Mission".

## UI

See https://github.com/DesignSafe-CI/DS-User-Guide/pull/60.
